with-development-client/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "with-development-client",
+  "description": "This bare project template includes a minimal setup for using unimodules, expo-dev-menu and expo-dev-launcher with React Native.",
+  "version": "40.0.0",
+  "main": "index.js",
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "web": "expo start --web",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "expo": "~40.0.0-beta.0",
```

Comment:
should update this version to the latest version on the sdk-40 branch https://github.com/expo/expo/blob/sdk-40/templates/expo-template-bare-typescript/package.json#L14